e2e/pages/adf/contentServicesPage.ts

```diff
@@ -59,13 +59,102 @@ export class ContentServicesPage {
     searchInputElement = element(by.css('input[data-automation-id="content-node-selector-search-input"]'));
     shareNodeButton = element(by.cssContainingText('mat-icon', ' share '));
 
-    getElementsDisplayed() {
+    async getNodesDisplayed(alfrescoJsApi) {
+
+        let promises = [];
+        let nodeList;
+
+        let idList = await this.getElementsDisplayedId();
+        let numberOfElements = await this.numberOfResultsDisplayed();
+        for (let i = 0; i < (numberOfElements - 1) ; i++ ) {
+            promises.push(alfrescoJsApi.core.nodesApi.getNode(idList[i]));
+        }
+        nodeList = await Promise.all(promises);
+        return nodeList;
+    }
+
+    getElementsDisplayedCreated() {
         let deferred = protractor.promise.defer();
-        let fileNameLocator = by.css("div[id*='document-list-container'] div[class*='adf-datatable-row'] div[title='Display name'] span[class='adf-datatable-cell-value']");
+        let fileNameLocator = by.css("div[id*='document-list-container'] div[class*='adf-datatable-row'] div[title='Created'] span");
```

Comment:
We shouldn't have document-list locators inside this demo-shell page. Maybe you could move this locator and the body of this method inside contentList(which is the PO for document-list) and then use it in here. [Keep in mind that the name of the column must remain on demo-shell page]. Same comment for below methods.